sphinx_gallery/gen_rst.py

```diff
@@ -647,6 +641,11 @@ def generate_file_rst(fname, target_dir, src_dir, gallery_conf):
                 example_rst += bcontent + '\n'
                 example_nb.add_markdown_cell(text2string(bcontent))
 
+    # Writes md5 checksum if example has build correctly
+    if not src_file in gallery_conf['failing_examples']:
+        with open(example_file + '.md5', 'w') as file_checksum:
+            file_checksum.write(get_md5sum(example_file))
```

Comment:
I'd rather do that at the same place where the code example is run. It is a bit weird to check in gallery_conf['failing_examples'] later ...